hyfetch/distros/distro_detector.py

```diff
@@ -123,6 +123,10 @@ def detect(name: str) -> AsciiArt | None:
     if name.startswith('arcolinux'):
         from .arcolinux import arcolinux
         return arcolinux
+
+    if name.startswith('furreto'):
```

Comment:
This file and everything in this folder get auto-generated by [this script](https://github.com/hykilpikonna/hyfetch/blob/master/tools/list_distros.py). It _shouldn't_ cause problems to modify like this, but it will be overwritten every time a release happens, and doesn't really impact functionality till then.

Reply:
wait so i shouldn't have touched that file then or..?

Reply:
It wasn't bad to, but also wasn't necessary.